components/admin.tsx

```diff
@@ -0,0 +1,1066 @@
+							>
+								Next
+							</Button>
+						</div>
+					</div>
+				</TabsContent>
+				<TabsContent value="ceremonies">
+					<div className="rounded-md border">
+						<Table>
+							<TableHeader>
+								{ceremonyTable.getHeaderGroups().map((headerGroup) => (
+									<TableRow key={headerGroup.id}>
+										{headerGroup.headers.map((header) => {
+											return (
+												<TableHead key={header.id}>
+													{header.isPlaceholder
+														? null
+														: flexRender(
+															header.column.columnDef.header,
+															header.getContext(),
+														)}
+												</TableHead>
+											);
+										})}
+									</TableRow>
+								))}
+							</TableHeader>
+							<TableBody>
+								{ceremonyTable.getRowModel().rows?.length ? (
+									ceremonyTable.getRowModel().rows.map((row) => (
+										<TableRow
+											key={row.id}
+										>
+											{row.getVisibleCells().map((cell) => (
+												<TableCell key={cell.id}>
+													{flexRender(
+														cell.column.columnDef.cell,
+														cell.getContext(),
+													)}
+												</TableCell>
+											))}
+										</TableRow>
+									))
+								) : (
+									<TableRow>
+										<TableCell
+											colSpan={ceremonyColumns.length}
+											className="h-24 text-center"
+										>
+											No results.
+										</TableCell>
+									</TableRow>
+								)}
+							</TableBody>
+						</Table>
+					</div>
+					<div className="flex items-center justify-end space-x-2 py-4">
+						<div className="flex-1 text-sm text-muted-foreground">
+							Page {ceremonyTable.getState().pagination.pageIndex + 1} of{" "}
+							{ceremonyTable.getPageCount()}
+						</div>
+						<div className="space-x-2">
+							<Button
+								variant="outline"
+								size="sm"
+								onClick={() => ceremonyTable.previousPage()}
+								disabled={!ceremonyTable.getCanPreviousPage()}
+							>
+								Previous
+							</Button>
+							<Button
+								variant="outline"
+								size="sm"
+								onClick={() => ceremonyTable.nextPage()}
+								disabled={!ceremonyTable.getCanNextPage()}
+							>
+								Next
+							</Button>
+						</div>
+					</div>
+					<div className="grid grid-cols-3 gap-5 lg:gap-3 w-full max-w-4xl auto-cols-auto">
+						<div className="grid-cols-subgrid">
+							<Card className="mx-auto">
+								<CardHeader>
+									<CardTitle>Add a Ceremony</CardTitle>
+									<CardDescription>
+										Create a new Passport Ceremony. The ceremony will appear for
+										selection once created.
+									</CardDescription>
+								</CardHeader>
+								<CardContent>
+									<Form {...createForm}>
+										<form
+											onSubmit={createForm.handleSubmit(onSubmitCreate)}
+											className="space-y-8"
+										>
+											<FormField
+												control={createForm.control}
+												name="new_ceremony_time"
```

Comment:
any reason why these are snake_case? feels out of place

Reply:
Resolved in [b9bfab8](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/b9bfab814516ac0d795573f35a8422e1878bf802)